### PR TITLE
Fix mismatching labels and values

### PR DESCRIPTION
**Pull request by berroar**
### What is the context of this PR?
This PR has been raised to fix instances in schemas where answer labels do not match answer values. For more info see [Trello](https://trello.com/c/rXoPEdks/3367-review-schemas-for-value-and-labels-mismatching).

The changes made here need to be implemented before the rule can be added to the schema validator - [PR here](https://github.com/ONSdigital/eq-schema-validator/pull/172).

I have also updated the test_schemas.sh script so that it now only checks schemas in data/en as generated schemas in other languages will no longer pass validation (answer labels are translated but values are not).

### How to review 
Run the schema validation tests using the '3367-review-schemas-for-value-and-labels-mismatching' branch of the eq-schema-validator and check that all tests are being passed.

Check that the changes made here don't break the functionality of the existing schemas.

### CODE REVIEW

**Comment by berroar** [comment]
[Review schemas for value and labels mismatching](https://trello.com/c/rXoPEdks/3367-review-schemas-for-value-and-labels-mismatching)

File: data-source/jsonnet/england-wales/ccs/blocks/individual/past_usual_household_address.jsonnet [hunk 1]
```diff
@@ -17,7 +17,7 @@ local question(title) = {
               placeholders.address,
             ],
           },
-          value: 'household-address',
```
Comment (ajmaddaford) [inline data-source/jsonnet/england-wales/ccs/blocks/individual/past_usual_household_address.jsonnet]: This should be `{household_address}`. This applies to all places this occurs.

File: data-source/jsonnet/england-wales/individual/blocks/personal-details/confirm_dob.jsonnet [hunk 2]
```diff
@@ -95,7 +100,19 @@ local proxyNoValue = 'No, I need to change their date of birth';
...
+      },
+    },
+    {
+      goto: {
+        block: 'date-of-birth',
+        when: [
+          {
+            id: 'confirm-date-of-birth-answer',
+            condition: 'equals',
+            value: 'No, I need to change their date of birth',
```
Comment (ajmaddaford) [inline data-source/jsonnet/england-wales/individual/blocks/personal-details/confirm_dob.jsonnet]: - This routing rule can be folded into the original one with a `equals any` condition.
- The value strings should reference the variables `nonProxyNoValue` and `proxyNoValue`.

File: data-source/jsonnet/england-wales/individual/blocks/personal-details/confirm_dob.jsonnet [hunk 3]
```diff
@@ -1,7 +1,7 @@
 local placeholders = import '../../../lib/placeholders.libsonnet';
 local rules = import 'rules.libsonnet';
 
-local question(title, yesLabel, noLabel, noValue) = {
+local question(title, yesLabel, noLabel, noValue, yesValue) = {
```
Comment (ajmaddaford) [inline data-source/jsonnet/england-wales/individual/blocks/personal-details/confirm_dob.jsonnet]: The ordering of the variables is not consistent. It should be `yesLabel, yesValue, noLabel, noValue`.

File: data-source/jsonnet/england-wales/individual/blocks/personal-details/term_time_location.jsonnet [hunk 4]
```diff
@@ -103,7 +103,7 @@ local otherNonUkAddressOptions = {
           },
         ],
       },
-      value: '30-day-address',
+      value: 'The address in {country}',
```
Comment (ajmaddaford) [inline data-source/jsonnet/england-wales/individual/blocks/personal-details/term_time_location.jsonnet]: Should this be `The address in {thirty_day_address_country}`?

File: data-source/jsonnet/northern-ireland/individual/blocks/personal-details/confirm_dob.jsonnet [hunk 5]
```diff
@@ -95,7 +98,19 @@ local proxyNoValue = 'No, I need to change their date of birth';
...
+      },
+    },
+    {
+      goto: {
+        block: 'date-of-birth',
+        when: [
+          {
+            id: 'confirm-date-of-birth-answer',
+            condition: 'equals',
+            value: 'No, I need to change their date of birth',
```
Comment (ajmaddaford) [inline data-source/jsonnet/northern-ireland/individual/blocks/personal-details/confirm_dob.jsonnet]: Same comments for this file as the england-wales one.

**Pull request merged**